client/coral-admin/src/containers/ModerationQueue/ModerationContainer.js

```diff
@@ -70,8 +70,10 @@ class ModerationContainer extends Component {
       <div>
         <ModerationHeader asset={asset} />
         <ModerationMenu
-          activeTab={activeTab}
           asset={asset}
+          premodCount={data.premod.length}
```

Comment:
i know, but this pr is just UI. we'll pass different ints here when pagination is added.